alg/gdal_interpolateatpoint.cpp

```diff
@@ -17,6 +17,8 @@
 
 #include "gdalresamplingkernels.h"
 
+#include "gdal_vectorx_templ.h"
```

Comment:
what about shortening that to "gdal_vectorx.h" ? The template thing is just an implementation detail that a user using gdal::Vector2i wouldn't notice

Reply:
Done.

BTW, if there is a better location for this file than the folder `alg`, I can move it.

Reply:
Would probably be better in gcore/